examples/apps/scala/pom.xml

```diff
@@ -14,24 +14,24 @@
     <properties>
         <encoding>UTF-8</encoding>
         <xgboost.version>1.0.0-SNAPSHOT</xgboost.version>
-        <cudf.version>0.13-SNAPSHOT</cudf.version>
-        <spark.version>3.0.0-preview2</spark.version>
+        <cudf.version>0.14</cudf.version>
+        <spark.version>3.0.0</spark.version>
         <scala.version>2.12.8</scala.version>
         <scala.binary.version>2.12</scala.binary.version>
-        <cuda.classifier></cuda.classifier>
+        <cuda.classifier>cuda10-1</cuda.classifier>
     </properties>
 
     <repositories>
         <repository>
-            <id>spark.sw.nvidia.com</id>
+            <id>snapshots</id>
             <name>sw-spark-maven</name>
-            <url>https://gpuwa.nvidia.com/artifactory/sw-spark-maven</url>
+            <url>https://urm.nvidia.com:443/artifactory/sw-spark-maven</url>
```

Comment:
Should we remove this urm and use maven central for customer building?

Reply:
updated